### ISSUE TITLE
Create a gift card while buying a gift-card product

### ISSUE THREAD

**Issue by sahnhi** [Setono/SyliusGiftCardPlugin#114]
Hello,

I created a gift card product and followed the payment process, but once at the end I do not have my gift card with the code that is generated.
Should I call a road at the end with my parameters ? If yes which one ?

Thanks

**Comment by loevgaard** [comment]
What do you mean by 'but once at the end I do not have my gift card with the code that is generated'?

When you buy a gift card it is emailed to you (assuming emailing is enabled). You should also be able to see it in the admin area under 'Gift cards'

**Comment by GuillaumeTediber** [comment]
Hello @loevgaard ,

After creating the gift card product, we have added one variant.
We are using a custom ShopApi, we have added the variant to the cart and then we have paid it using Sylius state machine.
It seems that no gift card have been created do we have to create it after the 'pay' Sylius Payment's state machine action ?

We have not seen any triggers with 'Gift Card' product.

Thank you for your Help,
